blockchain/build-docker.sh

```diff
@@ -14,5 +14,7 @@ docker image build --target antelope_blockchain . -f ./blockchain/Dockerfile --f
 ./build-contracts.sh
 
 # Create docker image
-docker rm -f tonomy_blockchain_initialized || true
-docker image build --target tonomy_blockchain_initialized . -f ./blockchain/Dockerfile --force-rm -t tonomy_blockchain_initialized
+docker rm -f tonomy_blockchain_initialized || true 
+docker build -t easycleos --target tonomy_easycleos . -f ./blockchain/Dockerfile
```

Comment:
docker image build --target tonomy_blockchain_initialized . -f ./blockchain/Dockerfile --force-rm -t tonomy_blockchain_initialized
docker image build --target tonomy_blockchain_easycleos . -f ./blockchain/Dockerfile --force-rm -t tonomy_blockchain_easycleos

Reply:
- keey naming consistent
- build the targets in order